### PR TITLE
Enable parsing of merkleblock messages

### PR DESCRIPTION
**Pull request by langerhans**
Needs to construct the parts of the FilteredBlock manually at this time, but has potential for optimzations.

### CODE REVIEW

File: src/main/java/org/libdohj/core/AltcoinSerializer.java [hunk 1]
```diff
@@ -1,18 +1,14 @@
-    /*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
+/*
+* To change this license header, choose License Headers in Project Properties.
```
Comment (rnicoll) [inline src/main/java/org/libdohj/core/AltcoinSerializer.java]: Sorry, just spotted this; happy with the first line indentation fix, but could you put the space back in the rest so they line up please?

Comment (rnicoll) [inline src/main/java/org/libdohj/core/AltcoinSerializer.java]: Or, actually, just rip it out as not really useful

Comment (langerhans) [inline src/main/java/org/libdohj/core/AltcoinSerializer.java]: Sorry, that was my IDE's auto-formatter. Removed the template comment.

**Comment by rnicoll** [comment]
Thanks!

**Pull request merged**